### ISSUE TITLE
application results

### ISSUE THREAD

**Issue by yr18** [mveres01/pytorch-drl4vrp#6]
Hi,
Sorry to raise an issue again.

I ran this code with checkpoint for vrp20 with load 30 and max demand 9 however the best result is 827(unit) for case A-N33-K5(optimal result is 661) most heuristic algorithm can get that result.  

A-N33-K5 is a standard instance which has 33 nodes and vehicle capacity is 100.
you can download the case from this website. http://www.bernabe.dorronsoro.es/vrp/   I scaled down the coordinates to 0,1 range to avoid large numbers.

I tried a trained vrp37 weight with load 44 however the result is even worse.

In the application part I used 1 for the train-size and valid-size.  I also tried A-N60-K9 but the result is 1943 and it is also far away from the optimal result( 1354)(I used the trained weight for VRP50)

I wonder how I can improve the result? Change the weight?Retrain the model? Is it due to the batch-size or learning rate? Or the model is not robust enough? ( I got a pretty good result when using the vrp20 weight on a vrp37 real case problem. I used to think the model is quite robust.)

Thank you so much for your time and help.

**Comment by mveres01** [comment]
From a quick glance, the major thing that sticks out is that you aren't quite matching the problem domains. 

Having a load of 30 and demand of 9 (vrp20) versus a load of 100 and demand of 24 (A-N33-K5, I think 24 was max) means that the vehicle is going to have to make extra trips to the depot, which will increase the tour length. It's subtle, but if you want to use a pre-trained vrp20, I would try to match the conditions more closely by increasing the max_load to 37.5 & keeping max_demand at 9. 

Though what would be even better would be to train a model that matches those conditions directly (i.e. VRP 33, load 100, demand 24). 

The authors have also open-sourced their [work ](https://github.com/OptMLGroup/VRP-RL) since this repo has gone live that's probably worth looking into

**Comment by yr18** [comment]
Hi,

I tried the pretrained model as close as to the problem case however the result did not get better.

I do not think tensorflow version could improve this problem a lot cause you posed some comparison there. They are similar.

Is it possible to increase the learning ability of this algorithm ?
I wonder if there is some way to help the model learn the pattern better. A deeper network?
